src/applications/mhv/medications/sass/medications.scss

```diff
@@ -55,6 +55,13 @@
   display: flex;
   justify-content: center;
 }
+.routes-container {
+  @media (max-width: $small-screen) {
+    // .vads-l-grid-container adds 15px on left and right, this overwrites that
```

Comment:
assuming something like: `small-screen:vads-u-padding-x--1 ` didn't work here? Did you try using it after the `vads-l-grid-container`?

Reply:
yeah i did try that, it didnt work. It seems like the vads-l-grid-container 's spacing took precedence.